package-lock.json

```diff
@@ -6751,12 +6751,20 @@
       }
     },
     "eslint-plugin-flowtype": {
-      "version": "2.35.1",
-      "resolved": "https://registry.npmjs.org/eslint-plugin-flowtype/-/eslint-plugin-flowtype-2.35.1.tgz",
-      "integrity": "sha512-YTCeVsMOi3ga8PJjdAV97FaTNXNknzrO+4ZDMHJN65i4uMjL5KgfgQZpyVsBirWOfgXMKRduxpsyM64K/0LbXw==",
+      "version": "4.3.0",
```

Comment:
I noticed you updated `<SearchBar />` to use the more up to date flow declarations to use commas instead of semicolons. Will all the existing files using the legacy semicolon declarations cause linting errors? PLAT-44570 updates all dependencies. If this causes errors when you run `npm run lint`, it might be better to wait and just use semicolons here for now.